test/app.test.js

```diff
@@ -0,0 +1,9 @@
+/* global test,expect true */
```

Comment:
Haha, sorry I forgot that underscores bold in markdown. I meant to say that just looks for tests in a folder called `__tests__` by default.